### PR TITLE
K8SPXC-480 in case of update use '33062' port when proxy protocol is …

### PR DESCRIPTION
**Pull request by hors**
[K8SPXC-480](https://jira.percona.com/browse/K8SPXC-480)

…enabled

### CODE REVIEW

File: pkg/controller/pxc/upgrade.go [hunk 1]
```diff
@@ -329,7 +330,12 @@ func (r *ReconcilePerconaXtraDBCluster) proxyDB(cr *api.PerconaXtraDBCluster) (q
 		host = fmt.Sprintf("%s-haproxy.%s", cr.ObjectMeta.Name, cr.Namespace)
 		proxySize = cr.Spec.HAProxy.Size
 
-		if cr.CompareVersionWith("1.6.0") >= 0 {
+		hasKey, err := cr.ConfigHasKey("mysqld", "proxy_protocol_networks")
+		if err != nil {
+			return database, errors.Wrap(err, "check if congfig has proxy_protocol_networks key")
```
Comment (tplavcic) [inline pkg/controller/pxc/upgrade.go:335]: typo `check if congfig` -> `check if config`

**Review by tplavcic — CHANGES_REQUESTED** [review]

File: pkg/controller/pxc/upgrade.go (hunk 1 above)
Comment (delgod) [inline pkg/controller/pxc/upgrade.go:335]:
```suggestion
			return database, errors.Wrap(err, "check if config has proxy_protocol_networks key")
```

**Review by delgod — CHANGES_REQUESTED** [review]

**Review by delgod — APPROVED** [review]

**Review by tplavcic — APPROVED** [review]

**Review by nonemax — APPROVED** [review]

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| haproxy | passed |
| scaling | passed |
| big-data | passed |
| recreate | passed |
| storage | passed |
| self-healing | passed |
| init-deploy | passed |
| scaling-proxysql | passed |
| upgrade | passed |
| restore-to-encrypted-cluster | passed |
| limits | passed |
| self-healing-advanced | passed |
| monitoring | passed |
| monitoring-2-0 | passed |
| demand-backup | passed |
| upgrade-consistency | passed |
| operator-self-healing | passed |
| affinity | passed |
| one-pod | passed |
| security-context | passed |
| auto-tuning | passed |
| proxysql-sidecar-res-limits | passed |
| users | passed |
| scheduled-backup | failed |
| tls-issue-self | passed |
| tls-issue-cert-manager | passed |
| tls-issue-cert-manager-ref | passed |
| demand-backup-encrypted-with-tls | passed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| recreate | passed |
| scaling | passed |
| haproxy | passed |
| big-data | passed |
| storage | passed |
| scaling-proxysql | passed |
| init-deploy | passed |
| self-healing | passed |
| upgrade | passed |
| restore-to-encrypted-cluster | passed |
| limits | passed |
| upgrade-consistency | passed |
| self-healing-advanced | passed |
| monitoring | passed |
| monitoring-2-0 | passed |
| operator-self-healing | passed |
| security-context | passed |
| affinity | passed |
| one-pod | passed |
| auto-tuning | passed |
| proxysql-sidecar-res-limits | passed |
| users | passed |
| demand-backup | passed |
| tls-issue-self | passed |
| scheduled-backup | failed |
| tls-issue-cert-manager | passed |
| tls-issue-cert-manager-ref | passed |
| demand-backup-encrypted-with-tls | failed |

**Pull request merged**